### PR TITLE
add index to speed up scheduler states query

### PR DESCRIPTION
**Pull request by AlexSJ1990**

### CODE REVIEW

**Review by FredrikAreschoug — APPROVED** [review]

File: src/main/resources/db/changelog/migrations/20201016-1111-add-scheduler-states-member-id-created-at-index.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+
+--changeset alex:20201016-1111-add-scheduler-states-member-id-created-at-index
+
+CREATE INDEX scheduler_states_member_id_changed_at_idx ON scheduler_states(member_id, changed_at desc)
```
Comment (vonElfvin) [inline src/main/resources/db/changelog/migrations/20201016-1111-add-scheduler-states-member-id-created-at-index.sql:5]: Why idx?
```suggestion
CREATE INDEX scheduler_states_member_id_changed_at ON scheduler_states(member_id, changed_at desc)
```

Comment (vonElfvin) [reply src/main/resources/db/changelog/migrations/20201016-1111-add-scheduler-states-member-id-created-at-index.sql:5]: image
The doesn't look the same

Comment (vonElfvin) [reply src/main/resources/db/changelog/migrations/20201016-1111-add-scheduler-states-member-id-created-at-index.sql:5]: I think `CONCURRENTLY` is good to have as well

Comment (johantjellden) [reply src/main/resources/db/changelog/migrations/20201016-1111-add-scheduler-states-member-id-created-at-index.sql]: With regards to the "idx" postfix.
We dont have a naming standard for our databases yet. For this case look at the other indexes in account-service and make sure we follow that standard.

**Review by johantjellden — APPROVED** [review]

**Pull request merged**